alacritty_terminal/src/config/colors.rs

```diff
@@ -16,6 +16,24 @@ pub struct Colors {
     pub dim: Option<DimColors>,
     pub indexed_colors: Vec<IndexedColor>,
     pub search: SearchColors,
+    pub opaque_background_colors: bool,
+}
+
+impl Default for Colors {
+    fn default() -> Self {
+        Self {
+            primary: Default::default(),
+            cursor: Default::default(),
+            vi_mode_cursor: Default::default(),
+            selection: Default::default(),
+            normal: Default::default(),
+            bright: Default::default(),
+            dim: Default::default(),
+            indexed_colors: Default::default(),
+            search: Default::default(),
+            opaque_background_colors: true,
```

Comment:
```suggestion
            opaque_background_colors: true,
            primary: Default::default(),
            cursor: Default::default(),
            vi_mode_cursor: Default::default(),
            selection: Default::default(),
            normal: Default::default(),
            bright: Default::default(),
            dim: Default::default(),
            indexed_colors: Default::default(),
            search: Default::default(),
```

Putting non-default fields at the top makes it easy to see why there is a custom `Default` implementation.